pertpy/tools/_distances/_distances.py

```diff
@@ -857,3 +907,94 @@ def onesided_distances(
 
     def from_precomputed(self, P: np.ndarray, idx: np.ndarray, **kwargs) -> float:
         raise NotImplementedError("ClassifierClassProjection cannot be called on a pairwise distance matrix.")
+
+
+class MeanVarDistnDistance(AbstractDistance):
+    """
+    Distance betweenv mean-var distributions of gene expression.
```

Comment:
Move this up to the line above and end the docstring there. We don't need empty lines